src/mjolnir/ingest_transit.cc

```diff
@@ -0,0 +1,790 @@
+        GraphId currId = GraphId(tile_level.tiles.TileId(y, x), 3, 0);
+
+        tileTransitInfo currTile;
+        if (tile_map.find(currId) != tile_map.end()) {
+          currTile = tile_map[currId];
+        } else {
+          currTile.graphId = currId;
+        }
+        currTile.tile_stops.insert({currStopId, feed_path});
+        gtfs::StopTimes currStopTimes = feed.get_stop_times_for_stop(currStopId);
+
+        for (const auto& stopTime : currStopTimes) {
+          // add trip, route, agency and service_id from stop_time
+          // could use std::set to not check if it already exists
+
+          // use a set instead to keep track of these points
+          if (stopTime.trip_id.empty() || !feed.get_trip(stopTime.trip_id)) {
+            continue;
+          }
+          gtfs::Trip currTrip = *(feed.get_trip(stopTime.trip_id));
+          if (currTrip.service_id.empty() || currTrip.route_id.empty() ||
+              !feed.get_route(currTrip.route_id)) {
+            continue;
+          }
+          currTile.tile_services.insert({currTrip.service_id, feed_path});
+          currTile.tile_trips.insert({currTrip.trip_id, feed_path});
+
+          currTile.tile_shapes.insert({currTrip.shape_id, feed_path});
+          // remember the shape id such that we can access it later
+          gtfs::Route currRoute = *(feed.get_route(currTrip.route_id));
+          currTile.tile_routes.insert({currRoute.route_id, feed_path});
+          currTile.tile_agencies.insert({currRoute.agency_id, feed_path});
+          // add to the queue of tiles
+        }
+        tile_map[currId] = currTile;
+      }
+    }
+  }
+  std::priority_queue<tileTransitInfo> prioritized;
+  for (auto it = tile_map.begin(); it != tile_map.end(); it++) {
+    prioritized.push(it->second);
+  }
+  return prioritized;
+}
+
+std::unordered_map<gtfs::Id, GraphId> write_stops(Transit& tile, const tileTransitInfo& tile_info) {
+
+  const auto& tile_stopIds = tile_info.tile_stops;
+  auto node_id = tile_info.graphId;
+  feedCache stopFeeds;
+
+  std::unordered_map<gtfs::Id, GraphId> stop_graphIds;
+  // loop through all stops inside the tile
+  for (const feedObject& feed_stop : tile_stopIds) {
+    const auto& tile_stopId = feed_stop.id;
+    const auto& feed = stopFeeds(feed_stop);
+    auto* node = tile.add_nodes();
+    gtfs::Stop tile_stop = *(feed.get_stop(tile_stopId));
+    node->set_lon(tile_stop.stop_lon);
+    node->set_lat(tile_stop.stop_lat);
+    node->set_type(static_cast<int>(tile_stop.location_type));
+    node->set_graphid(node_id);
+    node_id++;
+    // node->set_prev_graphid(); what is prev?
+    node->set_name(tile_stop.stop_name);
+    // node->set_osm_way_id(); where is this?
+    node->set_timezone(tile_stop.stop_timezone);
+    // 0 is No Info ; 1 is True ; 2 is False
+    bool wheelchair_accessible = (tile_stop.wheelchair_boarding == "1");
+    node->set_wheelchair_boarding(wheelchair_accessible);
+    // node->set_generated(feed_stop.generated); ?
```

Comment:
i this is a value add from transitland: https://www.transit.land/documentation/datastore/stops#generated-platforms-and-egresses

basically if there is a complex station you want to know the entry and exit points of it to connect it to the road network. some feeds dont have this information and indeed some stations/stops are trivial (bus stop along the road for example). in those cases one can generate an in/egress (if the downstream consumer, valhalla) expects there to always be in/egress information. what we can do is look in convert transit to see how we actually use it as i dont remember off the top of my head. not important for now tbh